### DISCUSSION TITLE
How do I trigger an hubot response inside a post request?

### DISCUSSION THREAD

**Discussion by rondigital1** [hubotio/hubot#1596] [Q&A] [answered]
Hi all,

How do I trigger a robot.respond() or a similiar action inside a robot.router.post() request?

Thanks

**Comment by technicalpickles** [comment, marked as answer]
You mean, you want to send a message in response to a route?

You can use `robot.messageRoom` described here: https://github.com/hubotio/hubot/blob/master/docs/scripting.md#messages-to-a-room-or-user . You would need to know a room to send it to.
